packages/components/src/components/tooltip/tooltip.js

```diff
@@ -78,6 +78,12 @@ const getMenuOffset = (menuBody, menuDirection) => {
   return undefined;
 };
 
+// @questions
+// -- What is the expected behavior if the user clicks on the trigger when the tooltip is already open
+// -- What's the focus state look like when there is only rich text within a tooltip (no buttons/links/etc)
+// -- There is a "flash" where the next element (another trigger in the demo) is focused before focus is
```

Comment:
Good question. I believe debouncing was for "keeping it open" behavior (see above) - Or even if it's not the case, it's just for achieving the intended interaction. So as long as the UI works as intended we can safely remove the debouncing logic.